mpxe/integration_tests/test_leds.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+import time
+
+from mpxe.integration_tests import int_test
+from mpxe.sims.controller_board_blinking_leds import Leds
+
+class TestLeds(int_test.IntTest):
+    def setUp(self):
+        super(TestLeds, self).setUp()
+        self.leds = self.manager.start('controller_board_blinking_leds', Leds())
+
+    def test_leds(self):
+        time.sleep(1)
```

Comment:
Same comment as previously, this doesn't actually seem to be testing anything?

Reply:
I think it's worth keeping if only as a simple demo to explain what MPXE is doing, very easy to relate it to what's going on in hardware. Since integration tests are going to be fairly lengthy anyways I think an extra second for this is fine